Feliz.Recharts/Line.fs

```diff
@@ -6,6 +6,14 @@ open Fable.Core
 
 [<Erase>]
 type line =
+    static member inline strokeWidth(value : int) = Interop.mkLineAttr "strokeWidth" value
+    /// The source data, in which each element is an object.
+    static member inline data (values: seq<'a>) = Interop.mkLineAttr "data" (Seq.toArray values)
+    /// The source data, in which each element is an object.
+    static member inline data (values: 'a list) = Interop.mkLineAttr "data" (List.toArray values)
+    /// The source data, in which each element is an object.
+    static member inline data (values: 'a array) = Interop.mkLineAttr "data" values
```

Comment:
I don't think the `data` attribute it supposed to go on the `line` type 🤔 only `dataKey` and the data goes on the `lineChart` type instead, see [here](https://github.com/Zaid-Ajaj/Feliz/blob/master/Feliz.Recharts/LineChart.fs#L18) Unless I am missing something from the docs at recharts, the `data` attrbute here should be removed

Reply:
@Zaid-Ajaj I think line also can have its own data:
http://recharts.org/en-US/examples/LineChartHasMultiSeries

`<Line dataKey="value" data={s.data} name={s.name} key={s.name} />`

Reply:
The link points to the latest documentation of recharts which is currently at v2.0.6 so this could be new functionality I haven't seen before in v1.8.6 which is what Feliz.Recharts depends on (1.8.6 >= version > 2.0.0) so if we add this here, we need to either: update Feliz.Recharts required recharts version and testing every sample against  latest 2.0.6 or just go back to older docs to see if the attribute was indeed available in v1.8.6 🤔 the former requires more work to figure whether there are any breaking changes in v2.x but I think that the API hasn't changed that much so that's a good sign

I hope that made sense, dependency management can be tricky 😅

Reply:
Does it help if I tell you that all the added functionality is working in my app? So, the line data should be working as well.

Reply:
Oh yeah absolutely! So if you used Femto to install the binding, then it means the data attribute is compatible. Ok cool, I will try to merge the PR later today and add a sample to docs to make sure everything works as expected. Thanks a lot again ❤️